internal/handlers/tigris/tigrisdb/errors.go

```diff
@@ -48,3 +48,18 @@ func IsAlreadyExists(err *driver.Error) bool {
 
 	return false
 }
+
+// IsInvalidArgument returns true if the error is "invalid argument" error.
+// This function is implemented to keep nolint in a single place.
+func IsInvalidArgument(err *driver.Error) bool {
+	if err == nil {
+		return false
+	}
+
+	//nolint:nosnakecase // Tigris named their const that way
+	if err.Code == api.Code_INVALID_ARGUMENT {
```

Comment:
I remember you had that `return err.Code == api.Code_INVALID_ARGUMENT`. Did you decide not to use it?

Reply:
Well, maybe later. Chasing a different problem now :)

Reply:
👌